screens/IntroAppScreen.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {
+  Image,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  TouchableHighlight
+} from 'react-native';
+import { WebBrowser } from 'expo';
+import HomeScreen from './HomeScreen';
+import CreateAccScreen from './CreateAccScreen';
+import { MonoText } from '../components/StyledText';
+
+export default class IntroAppScreen extends React.Component {
+  static navigationOptions = {
+    header: null,
+  };
+
+  render() {
+    return (    	
+      <View style={styles.container}>
+        <View style={[{ position: 'absolute', top: 0 }]}>
```

Comment:
This view seems to be repeated on multiple pages, is there a way to do this without code duplications?